nemo/collections/asr/models/ctc_models.py

```diff
@@ -196,6 +196,7 @@ def transcribe(
                     logits, logits_len, greedy_predictions = self.forward(
                         input_signal=test_batch[0].to(device), input_signal_length=test_batch[1].to(device)
                     )
+                    logits = logits.cpu()
```

Comment:
Can you add this to hybrid transcribe as well (CTC part)? And maybe other models that have transcribe implementation can benefit from this

Reply:
Sounds good, I have checked and fixed the same VRAM leak in hybrid and classification models too. I've also made the fix better for ctc_models so the cast back to cpu happens after decoding has run (so decoding can still gain benefit of GPU speed).